include/unifex/completion_channels.hpp

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) Rishabh Dwivedi <rishabhdwivedi17@gmail.com>
+ *
+ * Licensed under the Apache License Version 2.0 with LLVM Exceptions
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *   https://llvm.org/LICENSE.txt
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <unifex/detail/prologue.hpp>
+
+namespace unifex {
+struct set_value_t {};
+struct set_error_t {};
+struct set_done_t {};
```

Comment:
These should be the types of the `set_value`, `set_error`, and `set_done` CPOs, not unrelated structs.

Reply:
Hi @ericniebler Just made the changes you suggested. Please review it, if I have done the right changes or got something wrong.